README.md

```diff
@@ -13,42 +13,48 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
+Relays aggregate blocks from **multiple** builders in order to select the block with the highest fees. One instance of MEV-boost can be configured by a validator to connect to **multiple** relays. The consensus layer client of a validator proposes the most profitable block received from MEV-boost to the Ethereum network for attestation and block inclusion.
 
-Relays aggregate blocks from **multiple** builders in order to select the block with the highest fees. One instance of MEV-boost can be configured by a validator to connect to **multiple** relays. The Consensus Layer client of a validator proposes the most profitable block received from MEV-boost to the Ethereum network for attestation and block inclusion.
+A MEV-Boost security assessment was conducted on 2022-06-20 by [lotusbumi](https://github.com/lotusbumi). Additional audits of surrounding infrastructure, such as the Flashbots Relay, are currently underway. Additional information can be found in the [Security](#security) section of this repository.
 
 
-![mev-boost service integration overview](https://raw.githubusercontent.com/flashbots/mev-boost/main/docs/mev-boost-integration-overview.png)
+![MEV-Boost service integration overview](https://raw.githubusercontent.com/flashbots/mev-boost/main/docs/mev-boost-integration-overview.png)
 
 ## Who can run MEV-Boost?
 
 MEV-Boost is a piece of software that any PoS Ethereum node operator (including solo validators) can run as part of their Beacon Client software. It is compatible with any Ethereum consensus client. Support and installation instructions for each client can be found [here](#installing).
 
-
 ---
 
 See also:
 
 * [boost.flashbots.net](https://boost.flashbots.net)
-* [mev-boost Docker images](https://hub.docker.com/r/flashbots/mev-boost)
+* [MEV-Boost Docker images](https://hub.docker.com/r/flashbots/mev-boost)
 * [wiki](https://github.com/flashbots/mev-boost/wiki) & [troubleshooting guide](https://github.com/flashbots/mev-boost/wiki/Troubleshooting)
-* [mev-boost relay source code](https://github.com/flashbots/mev-boost-relay)
+* [MEV-Boost relay source code](https://github.com/flashbots/mev-boost-relay)
 * Specs:
   * [Builder API](https://ethereum.github.io/builder-specs)
   * [Flashbots Relay API](https://flashbots.notion.site/Relay-API-Spec-5fb0819366954962bc02e81cb33840f5)
 
-
-# Table of Contents
-
 - [Background](#background)
 - [Installing](#installing)
+  - [Binaries](#binaries)
+  - [From source](#from-source)
+  - [From Docker image](#from-docker-image)
+  - [Systemd configuration](#systemd-configuration)
+  - [Troubleshooting](#troubleshooting)
 - [Usage](#usage)
-- [The Plan](#the-plan)
+    - [Mainnet](#mainnet)
+    - [Goerli testnet](#goerli-testnet)
+    - [Sepolia testnet](#sepolia-testnet)
+      - [`test-cli`](#test-cli)
 - [API](#api)
 - [Maintainers](#maintainers)
 - [Contributing](#contributing)
 - [Security](#security)
+  - [Security Audits](#security-audits)
```

Comment:
why nested? may be outdated toc?

Reply:
toc is automatic, follows title markdown (## vs. ###)

Reply:
Automatic by whom? You can still remove/cleanup before committing?